scripts/seed_data.py

```
@@ -0,0 +1,114 @@
+    except FileNotFoundError:
+        print(f"❌ Error: {file_path} not found")
+        return
+    except json.JSONDecodeError:
+        print(f"❌ Error: Invalid JSON format in {file_path}")
+        return
+
+    try:
+        create_entities(data)
+        print("🎉 Data loaded successfully")
+    except (User.DoesNotExist, Classroom.DoesNotExist) as e:
+        print(f"❌ Entity not found: {str(e)}")
```

Ruff (0.8.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)